### PR TITLE
check we exist before creating.. not tested

### PR DESCRIPTION
**Pull request by paulgmiller**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -827,6 +827,9 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
     if location is None:
         location = rg_location
 
+    if client.get(resource_group_name, name):
+        raise CLIError('Cluster {} already exists in {}'.format(name, resource_group_name))
```
Comment (igagankalra) [inline src/aks-preview/azext_aks_preview/custom.py:831]:
```suggestion
        raise CLIError('Cluster {} already exists in {}'.format(name, resource_group_name))
```
Could write this as `raise CLIError(f'Cluster {name} already exists in {resource_group_name}')`
This is more readable and better way to substitute a String.

**Comment by yungezz** [comment]
hi @arrownj could you pls review the PR?

**Comment by arrownj** [comment]
Hi @paulgmiller, I can't quite understand what is PR for. Could you please help provide some description and refine the PR title to make it easy understanding ?

**Comment by yungezz** [comment]
hi @paulgmiller any update on this PR? thanks.

**Comment by yungezz** [comment]
hi @paulgmiller will close the PR for now, feel free to create new PR when ready, thanks.

**Pull request closed without merging**